surfer/viz.py

```diff
@@ -1745,6 +1751,39 @@ def set_distance(self, distance=None):
                 mlab.view(distance=distance, figure=f)
         return distance
 
+    def set_surf(self, surf):
+        """Change the surface geometry
+
+        Parameters
+        ----------
+        surf : str
+            freesurfer surface mesh name (ie 'white', 'inflated', etc.)
+        """
+        if self.surf == surf:
+            return
+
+        views = self._toggle_render(False)
+
+        # load new geometry
+        for geo in self.geo.values():
+            try:
+                geo.surf = surf
+                geo.load_geometry()
+            except IOError:  # surface file does not exist
+                geo.surf = self.surf
+                self._toggle_render(True)
+                raise
+
+        # update mesh objects (they use a reference to geo.coords)
+        with mayavi_threshold_patch:
+            for brain in self.brains:
+                brain.update_surf()
+
+        self.surf = surf
+        self._toggle_render(True, views)
+        for brain in self.brains:
+            brain._f.scene.reset_zoom()
```

Comment:
I wonder if this should be optional – but the geometries have slightly different centers, so without it, the brain seems to jump to the side.